projects/pt1/e2e_testing/main.py

```diff
@@ -71,10 +72,11 @@ def _get_argparse():
     parser.add_argument("-f", "--filter", default=".*", help="""
 Regular expression specifying which tests to include in this run.
 """)
-    parser.add_argument("-v", "--verbose",
-                        default=False,
-                        action="store_true",
-                        help="report test results with additional detail")
+    parser.add_argument("--log_level", default="WARNING", choices=["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"], help="set the log level")
```

Comment:
Found a tip for these choices at https://stackoverflow.com/a/71667675

add `type=str.upper` to the `parser.add_argument` options and then users can pass `--log_level=info` to have it be treated as `--log_level=INFO`